### DISCUSSION TITLE
Obtain the feature map from the last layer of the UNet network

### DISCUSSION THREAD

**Discussion by Guoxt** [Project-MONAI/MONAI#8072] [General]
I would like to obtain the feature map from the last layer of the UNet network implemented in monai.networks.nets.UNet(spatial_dims, in_channels, out_channels, channels, strides, kernel_size=3, up_kernel_size=3, num_res_units=0, act='PRELU', norm='INSTANCE', dropout=0.0, bias=True, adn_ordering='NDA'). How can I achieve this?

**Comment by KumoLiu** [comment]
Hi @Guoxt, please refer to this discussion: https://github.com/Project-MONAI/MONAI/discussions/5796

Hope it helps, thanks.
